plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ImportNativeAppEngineStandardProjectTest.java

```diff
@@ -146,7 +146,6 @@ private static void updateOldContainers(IProject project) throws CoreException {
         CloudToolsEclipseProjectUpdater.updateProject(project, SubMonitor.convert(null));
 
     ProjectUtils.waitForProjects(project);
-    ProjectUtils.waitUntilNoBuildErrors(project);
```

Comment:
We should move this check into `testImportAppEngineStandardJava8_from1_3_1`.  And perhaps incorporate a Java-7-error in `testImportAppEngineStandardJava7_from1_3_1` and `testImportProjectFrom131_oldStyleContainerWithAppEngineApi`?

https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/472424c9d545ce72098d29e1c2a5fff7e98264a6/plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/AppEngineWebXmlValidationTest.java#L63-L66

Reply:
How necessary is this? We have separate tests that cover that logic already and presumably it slows down the test and makes it more brittle.

Reply:
Well this is an integration test…. `waitForProjects()` waits until the build errors have stabilized so it shouldn't add any more time.

Reply:
OK, done.